x-pack/plugins/uptime/public/components/functional/monitor_list_actions_popover.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButtonIcon,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiPopover,
+  EuiText,
+  EuiToolTip,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { get } from 'lodash';
+import { LatestMonitor } from '../../../common/graphql/types';
+import { IntegrationLink } from './integration_link';
+
+interface MonitorListActionsPopoverProps {
+  basePath: string;
+  dateRangeStart: string;
+  dateRangeEnd: string;
+  monitor: LatestMonitor;
+}
+
+export const MonitorListActionsPopover = ({
+  basePath,
+  dateRangeStart,
+  dateRangeEnd,
+  monitor,
+  monitor: { ping },
+}: MonitorListActionsPopoverProps) => {
+  const popoverId = `${monitor.id.key}_popover`;
+  const [popoverIsVisible, setPopoverIsVisible] = useState<boolean>(false);
+  const domain = get(ping, 'url.domain', '');
+  return (
+    <EuiPopover
+      button={
+        <EuiButtonIcon
+          aria-label={i18n.translate(
+            'xpack.uptime.monitorList.observabilityIntegrationsColumn.popoverIconButton.ariaLabel',
+            {
+              defaultMessage: 'Opens integrations popover for monitor with url {monitorUrl}',
+              description:
+                'A message explaining that this button opens a popover with links to other apps for a given monitor',
+              values: { monitorUrl: monitor.id.url },
+            }
+          )}
+          color="subdued"
+          iconType="boxesHorizontal"
```

Comment:
Maybe there's a better icon option now that we don't show other icons next to this. The boxes make sense as an ellipsis to other icons, but alone they are weird. See the SS below:

![image](https://user-images.githubusercontent.com/131427/56991611-aff96b00-6b5d-11e9-843c-cb9c012701f2.png)

There is a popout icon, but that's not really right for a popover:

![image](https://user-images.githubusercontent.com/131427/56991666-d61f0b00-6b5d-11e9-9f69-29fa93e72f87.png)

Maybe we could get some input from @elastic/eui-design or @elastic/kibana-design ?

Reply:
@andrewvc I am fine with changing the icon. Would you be ok with me making a follow-up issue where we could discuss this in more detail so we don't hold up this PR any longer? There are other PRs downstream of this one that are blocked until some of this code gets merged.

Reply:
Yes, I'm OK with that